src/controllers/check.your.answers.controller.ts

```diff
@@ -26,12 +27,16 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
       const request: IExtensionRequest = sessionService.getRequest(req.chSession);
       const fullRequest: ExtensionFullRequest =
         await apiClient.getFullRequest(companyNumber, token, request.extension_request_id);
+      let companySummaryListRows = buildCompanySummaryListRows(companyInSession, true);
+      companySummaryListRows = companySummaryListRows.concat(
```

Comment:
Just wonder if this additional line of processing to change 'companySummaryListRows' a bit more could just be encapsulated into the 'buildCompanySummaryListRows()' function? Would mean passing 'req' into too...